### ISSUE TITLE
Investigate what would be required to use bitnami postgressql container 14.7.0

### ISSUE THREAD

**Issue by hardillb** [FlowFuse/helm#284]
### Description

https://hub.docker.com/r/bitnami/postgresql/tags?page=1&name=14.7.0

This would allow for installing the local postgresql database on ARM64.

**Comment by ppawlowski** [comment]
I ran our application with the bitnami's Postgres database deployed with the image `14.10.0-debian-11-r30` . 

Tests I executed:
* fresh install with `14.10.0-debian-11-r30` image + initial setup + create new project
* fresh install with `14.5.0-debian-11-r35` image + initial setup + create new project + update chart with postgres image set to `14.10.0-debian-11-r30` + add another project

Testing environment: 3-node Kubernetes cluster, x64 based, deployed in DigitalOcean.
No issues were discovered during tests.

My suggestion: upgrade the PostgreSQL version by setting up the `postgresql.image.tag` chart value to `14.10.0-debian-11-r30`. Upgrading subchart version to `12.0.0` is not possible since this version introduces PostgreSQL 15.
